.github/workflows/plugin_tests.yml

```diff
@@ -0,0 +1,36 @@
+name: Plugin TasksTimetable Tests
+
+on:
+  pull_request:
+  schedule:
+    - cron: '0 0 * * *'
+
+permissions:
+  actions: read
+  checks: none
+  contents: read
+  deployments: none
+  issues: read
+  packages: none
+  pull-requests: read
+  repository-projects: none
+  security-events: none
+  statuses: none
+concurrency:
+  group: ${{ github.head_ref }}
+  cancel-in-progress: true
+
+jobs:
+  PluginTests:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+    steps:
+      - name: run tests
+        uses: matomo-org/github-action-tests@v1
+        with:
+          git-head: ${{ github.head_ref }}
+          git-sha: ${{ github.event.pull_request.head.sha }}
+          plugin-name: 'TasksTimetable'
```

Comment:
Here I would have expected, not only passing the head/sha for the plugin repo to test with, but also the Matomo branch/tag to test against. Before that was handled with `PIWIK_TEST_TARGET`, which was `4.x-dev` for this plugin.
But for other tests than UI tests that was also used with the minimum required Matomo version (tag).

Reply:
added `test-target`